### DISCUSSION TITLE
PRC-77 tx & rx with 117/152?

### DISCUSSION THREAD

**Discussion by DoskalAridin** [IDI-Systems/acre2#1192] [Q&A] [answered]
should it not be possible to set a PRC-117 and PRC-152 to work with the PRC-77?
I've tried in game to set them on the same Freq, experimented if the waveform presets did anything, etc.
This is a niche use case, sure. but if it's something that could be done IRL, but it's that sort of attention to detail that I use ACRE2 over TFAR.

**Comment by rautamiekka** [comment]
What proof is there it works IRL ?

**Comment by Soldia1138** [comment, marked as answer]
It is possible. As per Documentation the PRC77 has Subtone Squelch on 150kHz. As the PRC152/117 can't set the CTCSS to 150kHz by default (other than the PRC148, which has that option), you could either set your presets to incorporate that frequency in the CTCSS List or you can simply disable the squelch. Set the Function Switch to Position "ON" for the PRC77 then.

**Reply by rautamiekka** [reply]
Right, now I remember reading off Wikipedia and some docs they're compatible (but how that's done, no clue), and the same is written on the ACRE wiki. So have you tested it in game ?

**Reply by DoskalAridin** [reply]
Verified, PRC77 needs function switch set to ON to receive. And PRC117 need it's analog squelch set to NOISE to receive PRC77 transmissions.
